spec/predicates/README.md

```diff
@@ -34,7 +34,7 @@ our [vetting process], and may be of general interest:
 [Release]: release.md
 [Runtime Traces]: runtime-trace.md
 [SCAI Report]: scai.md
-[Vulnerability]: vuln.md
+[Vulnerability]: vulns.md
```

Comment:
The other README uses the shorter name "VULNS", so it would be good to name these consistently. I do think that "Vulnerability" or even "Vulnerability Scan" would be clearer for the long name ("vulns" is still ok for the short predicate name).